src/main/java/com/enderio/base/common/blockentity/GraveHandler.java

```diff
@@ -32,6 +36,7 @@ public static void GraveDeath(LivingDropsEvent event) {
         }
         if (event.getEntity() instanceof Player player) {
             if (player.isCreative()) return;
+            if(!enableGrave.get()) return;
```

Comment:
move check inside the gamerules if and merge the first if statements together and putting each check on it's own line like:
if (entity == null
|| instanceof FakePlayer
...)
return;